src/main/java/org/cryptomator/cryptofs/dir/ExcludeDirIdBackupFilter.java

```diff
@@ -0,0 +1,16 @@
+package org.cryptomator.cryptofs.dir;
+
+import org.cryptomator.cryptofs.common.Constants;
+
+import java.io.IOException;
+import java.nio.file.DirectoryStream;
+import java.nio.file.Path;
+
+public class ExcludeDirIdBackupFilter implements DirectoryStream.Filter<Path> {
+
+	@Override
+	public boolean accept(Path entry) throws IOException {
+		return !entry.equals(entry.resolveSibling(Constants.DIR_BACKUP_FILE_NAME));
+	}
+
+}
```

Comment:
Can be a constant:

```suggestion
public interface DirListingFilters {

	static DirectoryStream.Filter<Path> EXCLUDE_DIR_ID_BACKUP = p -> !p.equals(p.resolveSibling(Constants.DIR_BACKUP_FILE_NAME));

}
```

Reply:
Resolved in https://github.com/cryptomator/cryptofs/pull/179/commits/d653636379b166eea02264eb7eeb765b3ae521ff